src/webview/android/main_pipe.rs

```diff
@@ -136,8 +154,35 @@ fn find_my_class<'a>(
   Ok(my_class.into())
 }
 
+fn set_background_color<'a>(
+  env: JNIEnv<'a>,
+  webview: JObject<'a>,
+  background_color: (u8, u8, u8, u8),
+) -> Result<(), JniError> {
+  let color_class = env.find_class("android/graphics/Color")?;
+  let color = env.call_static_method(
+    color_class,
+    "argb",
+    "(IIII)I",
+    &[
+      background_color.3.into(),
+      background_color.0.into(),
+      background_color.1.into(),
+      background_color.2.into(),
+    ],
+  )?;
+  env.call_method(webview, "setBackgroundColor", "(I)V", &[color])?;
+  Ok(())
+}
+
 #[derive(Debug)]
 pub enum WebViewMessage {
-  CreateWebView(String, bool),
+  CreateWebView {
+    url: String,
+    devtools: bool,
```

Comment:
we should take the entire webviewattributes struct here

Reply:
I don't think we can, there is bunch of handlers in WebViewAttributes that aren't send+sync

Reply:
well just create a dedicated struct then, otherwise this enum value will soon have 10 fields